nl_server/config.py

```diff
@@ -99,13 +104,30 @@ class EmbeddingsConfig:
   models: Dict[str, ModelConfig]
 
 
+#
+# Get Dict of vertex ai model to its info
+#
+def _get_vertex_ai_model_map() -> Dict[str, any]:
```

Comment:
nit: _get_vertex_ai_model_info() and use info instead of map below?

Reply:
done